src/excalidraw-app/index.tsx

```diff
@@ -625,13 +625,11 @@ const ExcalidrawWrapper = () => {
         </MainMenu.Item>
         <MainMenu.DefaultItems.Socials />
         <MainMenu.Separator />
-        <MainMenu.Group className="main-menu-footer">
-          <MainMenu.DefaultItems.ToggleTheme />
-          <MainMenu.Item className="language">
-            <LanguageList style={{ width: "100%" }} />
-          </MainMenu.Item>
-          <MainMenu.DefaultItems.ChangeCanvasBackground />
-        </MainMenu.Group>
+        <MainMenu.DefaultItems.ToggleTheme />
+        <div className="dropdown-menu-item-base" style={{ margin: "0.5rem 0" }}>
+          <LanguageList style={{ width: "100%" }} />
+        </div>
```

Comment:
I don't know how to abstract this wrapper yet, so I didn't.

One idea is to split `MainMenu.Item` into `MainMenu.ListItem` (the ones with `:hover` states, `onClick` handlers etc.) and `MainMenu.GenericItem`, latter of which would use the style of the div wrapper above. But honestly, the `GenericItem` isn't really generic item, it'd be just a container with some styling.

Another idea is to have different contexts for items. The list items would be the same `MainMenu.Item` components, but wrapped into a `<MainMenu.List>` which would style them as list items. But the issue here is that the styled div above again isn't an Item really, so we couldn't reuse the Item component for it as there aren't `onClick` handlers and such.

Reply:
what is `MainMenu.List` ,`MainMenu.ListItem`, `MainMenu.GenericItem` ? 
sounds too complex :p
Why do we need to abstarct this into a wrapper ?